NAMESPACE

```diff
@@ -130,6 +132,7 @@ export(showcase_bottom)
 export(showcase_left_center)
 export(showcase_top_right)
 export(sidebar)
+export(sidebar_open_on)
```

Comment:
If I were to vote, I think I would prefer to not have an exported function for this.

Reply:
I moved `sidebar_open_on()` to be used internally for now. We could easily export it down the line if we change our minds.